internal/qtls/go121.go

```diff
@@ -55,11 +55,9 @@ func SetupConfigForServer(qconf *QUICConfig, _ bool, getData func() []byte, acce
 	// add callbacks to save transport parameters into the session ticket
 	origWrapSession := conf.WrapSession
 	conf.WrapSession = func(cs tls.ConnectionState, state *tls.SessionState) ([]byte, error) {
-		// Add QUIC transport parameters if this is a 0-RTT packet.
-		// TODO(#3853): also save the RTT for non-0-RTT tickets
-		if state.EarlyData {
-			state.Extra = append(state.Extra, addExtraPrefix(getData()))
-		}
+		// Add QUIC transport parameters
+		state.Extra = append(state.Extra, addExtraPrefix(getData()))
```

Comment:
Is this also saving the transport parameters for non-0-RTT tickets?

Reply:
Yes, currently it is. I was indeed hesitant to whether only include the RTT in the non-0RTT session ticket or keep it the same with 0-RTT session ticket.

Reply:
It's better to have a small ticket. There's also less that can go wrong when we receive it.

We need to be careful when parsing the ticket that we don't attempt to parse the transport parameters when they're not there.

Reply:
Transport parameters are removed for non-0RTT tickets.